### PR TITLE
Added back long route warning

### PR DESCRIPTION
**Pull request by planecore**
If I remember correctly how it worked previously, it should work now again

### CODE REVIEW

File: app/screens/route-list/route-list-screen.tsx [hunk 1]
```diff
@@ -120,7 +120,7 @@ export const RouteListScreen = observer(function RouteListScreen({ navigation, r
           onRefresh={() => trains.refetch()}
           refreshing={trains.isRefetching}
           renderItem={renderRouteCard}
-          keyExtractor={(item) => item.trains[0]?.trainNumber.toString()}
+          keyExtractor={(item) => item.trains.map((train) => train.trainNumber).join()}
```
Comment (guytepper) [inline app/screens/route-list/route-list-screen.tsx:123]: Why this change was needed? 
Are there scenarios where the same train is being used for 2 different routes?

**Review by guytepper — APPROVED** [review]
Awesome! Thank you 🙏

File: app/screens/route-list/route-list-screen.tsx (hunk 1 above)
Comment (planecore) [reply app/screens/route-list/route-list-screen.tsx:123]: Yes, at one time the API gave me 2 routes with a same first train. Probably just a glitch at the Rail API but I think we need to support it (react threw me a duplicate key error)

Comment (guytepper) [reply app/screens/route-list/route-list-screen.tsx:123]: Interesting. Mind sharing what the route was?

Comment (planecore) [reply app/screens/route-list/route-list-screen.tsx:123]: Ben Gurion Airport to Be'er Sheva Center on 4/4/23 at 9:44.

Here's the API response:
image
